migration/src/main/resources/db/migration/V002__test-2.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS public.migrated2 (
+  persistence_id VARCHAR(255) NOT NULL,
+  sequence_number BIGINT NOT NULL,
+  created BIGINT NOT NULL,
+  snapshot BYTEA NOT NULL
+);
+CREATE TABLE test_user (
+  name VARCHAR(200)
+);
```

Comment:
We can move these files to the `test` folder instead. They are useful to test the migration tool itself.

Reply:
But these are meant to become the real migrations which we test with the tests. They should be part of what is tested, not of the tests.
We need to split those per supported database.

Reply:
Ok, I see you what your intention is now. 

I'm wondering how this will evolve in time. If we add some migration in 4.1.0, a user that started using the plugin in 4.0.0 won't have a migration table and the tool will apply all of them.
If we ship migrations in a jar, we must be sure that they are all idempotent. Easy to do in SQL, but those written in Java will need extra checks.